frontend/src/pages/workspace/profile/Index.tsx

```diff
@@ -0,0 +1,111 @@
+import { Container, Stack, Avatar, Typography, Button, FormControl } from "@mui/material";
+import { TextFieldElement, FormContainer } from "react-hook-form-mui";
+import { useSelector } from "react-redux";
+import { selectUser } from "../../../store/userSlice";
+import { useEffect, useMemo, useState } from "react";
+import { useDebounce } from "react-use";
+import { useUpdateUserNicknmaeMutation } from "../../../hooks/api/user";
+import { useCheckNameConflictQuery } from "../../../hooks/api/check";
+import { useGetWorkspaceQuery } from "../../../hooks/api/workspace";
+import { useParams } from "react-router-dom";
+const avatarSize = 117;
+
+function ProfileIndex() {
+	const params = useParams();
+	// eslint-disable-next-line
+	const { data: workspace } = useGetWorkspaceQuery(params.workspaceSlug);
```

Comment:
Please don't use `eslint-disable-next-line`.

I think that we can delete this two lines by moving to url `/profile`.

Reply:
thank you. i deleted it

[ee9e3ff](https://github.com/yorkie-team/codepair/pull/237/commits/ee9e3ffb4ec3d2803b62801d95a8838f45a571a8)